server/utils/arx_utils.py

```diff
@@ -768,15 +770,13 @@ def __get__(self, instance, owner):
         """
         return self.getter(owner)
 
-
 def get_full_url(url):
     """
     Gets the full url when given a partial, used for formatting links. For this to work
     properly, you should define your Site's url under the 'Sites' app in django's admin
     site.
     Args:
         url: A partial url from a few, like '/namespace/view/'
-
     Returns:
         A full url, like "http://www.example.com/namespace/view/"
     """
```

Comment:
Most of the whitespace changes violate PEP8. Spacing the docstrings I don't mind, but module-scoped functions are 2 spaces, methods are 1 space

Reply:
The whitespace changes are all artifacts of me cleaning up the merge and are only still there because they're super annoying to hunt down and delete.